src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,122 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import tasks.TaskList;
+
+/**
+ * Saves and retrieves tasks created by the user.
+ */
+public class Storage {
+
+    private final String FILENAME;
```

Comment:
i like that filename here is set as FINAL as we are only using a single file as local storage